parser/src/main/java/io/specmesh/apiparser/model/Bindings.java

```diff
@@ -18,16 +18,19 @@
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
 import lombok.NoArgsConstructor;
-import lombok.Value;
 import lombok.experimental.Accessors;
 
 /** Pojo representing a binding */
-@Value
+@Builder
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
Don't think these are needed for this PR. Not having them reduces the size of your API, which is a good thing.

Also, consider if this API lets the user build specs that are invalid.  This should be avoided. (I think lombok supports _required_ properties, which you should set)